javascript/javascript2/week2/hyfBay.js

```diff
@@ -0,0 +1,60 @@
+  
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
```

Comment:
This selector won't work, you can use `document.querySelector('#list')` instead.